tubemq-server/src/main/java/org/apache/tubemq/server/broker/web/BrokerAdminServlet.java

```diff
@@ -436,7 +436,7 @@ public void adminQuerySnapshotMessageSet(HttpServletRequest req,
         }
         final String topicName = (String) result.retData1;
         if (!WebParameterUtils.getIntParamValue(req,
-                WebFieldDef.PARTITIONID, false, -1, 0, result)) {
+                WebFieldDef.PARTITIONID, true, -1, 0, result)) {
```

Comment:
You should rebase master codes before submitting the code. 

Here, a modified bug was be rolled back: https://github.com/apache/incubator-inlong/pull/467

Reply:
This part is outdated view.

Reply:
I saw it